### ISSUE TITLE
Open sourcing

### ISSUE THREAD

**Issue by mmattamala** [leggedrobotics/wild_visual_navigation#282]
#### Software
- [x] Reworking the Configuration -> making the overlay less confusing and simple - throwing out the simple_parsing and instead using Omegaconfig + hydra @JonasFrey96 
- [ ] Allow to produce default slic segmentation and the version we published to RSS / Question @mmattamala what should we do here - remove all the fpr and scaling to make it look good ?
- [x] Jonas: Implement offline-pretrained model inference node / folder arbitrary image size crop network publish mask
- [ ] Jonas: Fix the confidence things - currently issue is still open

#### Additional features
- [x] Add ANYmal simulation configuration files in wvn_anymal
- [x] Can we add the simple description of anymal_d to the rosbags ?
- [ ] Reworking the dataset extraction using h5py would be quite nice for future projects using newly generated tool
- [ ] Integrate the offline learning and dataset extraction in a simpler manner

#### Data
- [x] Providing a 1min rosbag as an example @JonasFrey96 
- [x] Provide pre-trained checkpoints and 1 min rosbags
- [x] Jonas: Extract example images from 1min rosbags and put them in a folder for offline-pretrained model inference node

#### Simulation
- [ ] Add default cameras for the simulation - don't add the parameters to the default.yaml but instead have a separate file. Otherwise, when loading the default.yaml and other cameras to the param server errors can occur. I think with your delete params in the roslaunch, you may have resolved this, but I think the separate way is still clearer. 
- [x] Complete Jackal simulation with front camera

#### Installation
- [x] Create a docker image for this @mmattamala 
- [x] @mmattamala  virtual environment installation instructions
- [x] working launch files with the resizing of the images or exact conda instructions to run it in the hacky way - this may be not to bad actually with conda if we provide the PYTHONPATH  
- [x] Remove conda installation
- [x] Test full installation
- [ ] Add Piotr's code as a dependency once the repo is public for easier setup

#### Actions in additional dependencies
- [ ] Merge changes into elevation_mapping_cupy
- [ ] Coordinate with Piotr for open-sourcing his code

#### Testing
- [ ] Test different configuration settings
- [ ] Test smart carrot
- [ ] Test docker image
- [ ] Test anomaly detection by Robin alone
- [ ] Are all tests passing?

####  Documentation
- [ ] Create GIFs for readme
- [ ] Adding a drawing of the frames to the readme
- [ ] Update system figure @mmattamala 
- [ ] Add explanation of configuration in README
- [x] Add comments to the configuration

#### Project management and development
- [x] @mmattamala Remove stale branches, keep the minimum number possible
- [x] @mmattamala Clean up open PRS
- [x] Integrating testing into CI/CD ? 
- [x] Add watermarks/headers by ETH to the code
- [x] **Jonas** Add license 
- [ ] Remove all the issues before open-sourcing
- [x] **Jonas** check your own PRs and resolve them
- [x] **Jonas** Check orbit branch, move to new repo 
- [x] **Jonas** Verify if we infringe any license did we copy any code

**Comment by mmattamala** [comment]
Closing this since the repo is public now
